triage/2022-11-29.md

```diff
@@ -0,0 +1,183 @@
+# 2022-11-29 Triage Log
+
+A relatively quiet week for performance, with the notable exception of "Avoid
+`GenFuture` shim when compiling async constructs [#104321](https://github.com/rust-lang/rust/pull/104321)"
+ which brought sizeable wins on a number of stress test benchmarks. It probably
+won't be of huge benefit to most codebases, but should provide smaller wins to
+folks with large amounts of async-generated futures.
+
+Triage done by **@simulacrum**.
+Revision range: [a78c9bee4d9d51a3891bd8ecae1f28a93b83653b..8a09420ac48658cad726e0a6997687ceac4151e3](https://perf.rust-lang.org/?start=a78c9bee4d9d51a3891bd8ecae1f28a93b83653b&end=8a09420ac48658cad726e0a6997687ceac4151e3&absolute=false&stat=instructions%3Au)
```

Comment:
Nit: shorten the shas

Reply:
(My own approach to this is to extract the SHA from the url itself, where it won't be shortened, via a command like this in my script that I use to automate setting up triage each week: `PARENT=$(grep 'Revision range' $LAST_WEEK_FILE | sed -e 's/^.*end=\([0-9a-f]*\)&.*$/\1/')`)